docs/manual/developer/04_styleguide.md

```diff
@@ -0,0 +1,100 @@
+# Styleguide
+
+## All files
+* Use UNIX style line endings
+* One blank line at the end of the file
+
```

Comment:
Maybe some statement here about trailing white space in general?

"When not syntactically necessary, ..."